doc/examples/filters/plot_denoise.py

```diff
@@ -38,7 +38,7 @@
 noisy = astro + 0.6 * astro.std() * np.random.random(astro.shape)
```

Comment:
LOL I am definitely far from the expert here, but yes, it seems like that's what those functions are for! I would assume Gaussian noise would be more realistic than uniform noise, too...